translator.softHSM.Dockerfile

```diff
@@ -0,0 +1,74 @@
+#
+#   Docker Image for Translator Component in Test environments only.
+#   Translator depends on softHSM and OpenSC installations in Test environments.
+#
+FROM gradle:jdk11 AS build
+
+WORKDIR /app
+USER root
+ENV GRADLE_USER_HOME ~/.gradle
+
+COPY build.gradle build.gradle
+COPY settings.gradle settings.gradle
+COPY proxy-node-shared/ proxy-node-shared/
+
+ARG component=proxy-node-gateway
+COPY ${component}/src ${component}/src
+COPY ${component}/build.gradle ${component}/build.gradle
+
+RUN gradle --no-daemon --quiet -p ${component} --parallel install -x test
+ENTRYPOINT ["gradle", "--no-daemon"]
+
+# Runtime Image
+FROM openjdk:11-jre-slim
+
+USER root
+ENV OPEN_SC_VERSION=0.19.0
+
+# Install softHSM and OpenSC
+RUN apt-get update && apt-get install -y softhsm pcscd libccid libpcsclite-dev libssl-dev libreadline-dev autoconf automake build-essential docbook-xsl xsltproc libtool pkg-config wget && \
```

Comment:
Having all the commands compacted together will potentially make docker slower as it has todo all theses tasks every time the image is rebuilt.

Reply:
Have created a separate docker image for the softhsm installation and included it as the runtime image so that this installation does not have to happen every time there is a build.